File: 코딩테스트 입문/숫자/7의 개수.java
class Solution {
    public int solution(int[] array) {
        int answer = 0;
        for(int a : array) {
            String strA = Integer.toString(a);
            for(char s : strA.toCharArray()) {
                if(s == '7') {
                    answer++;
                }
            }
        }
        return answer;
    }
}

File: 프로그래머스/코딩테스트 입문/숫자/7의 개수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> array) {
    int answer = 0;
    for(auto a : array) {
        string strA = to_string(a);
        for(auto s : strA) {
            if(s == '7') {
                answer++;
            }
        }
    }
    return answer;
}